WTWSMS/history/provinces/100 - Avranches.txt

```diff
@@ -22,8 +22,6 @@ religion = celtic_pagan
 terrain = forest
 # History
 1.1.1 = { # Viking takeover
-	remove_settlement = b_caen
-	remove_settlement = b_bayeux
```

Comment:
I've found this in Wikipedia:
The regions that later became the Duchies of Normandy and Brittany each experienced devastating Viking raids, with Brittany occupied by Vikings from 907 to 937. In 933 Avranches and its territory, the Avranchin, were ceded to the Normans.